[2022-08-29 15:48:11,139] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-01-10T06:00:00+00:00 [queued]>
[2022-08-29 15:48:11,153] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-01-10T06:00:00+00:00 [queued]>
[2022-08-29 15:48:11,154] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 15:48:11,155] {taskinstance.py:1239} INFO - Starting attempt 28 of 28
[2022-08-29 15:48:11,156] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-29 15:48:11,171] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): entry_task> on 2021-01-10 06:00:00+00:00
[2022-08-29 15:48:11,185] {standard_task_runner.py:52} INFO - Started process 12131 to run task
[2022-08-29 15:48:11,197] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'h1b_lca_data_v3', 'entry_task', 'scheduled__2021-01-10T06:00:00+00:00', '--job-id', '1538', '--raw', '--subdir', 'DAGS_FOLDER/upload_h1b_data_to_gcp.py', '--cfg-path', '/tmp/tmp40zkzyqh', '--error-file', '/tmp/tmpmm_z_nc8']
[2022-08-29 15:48:11,214] {standard_task_runner.py:77} INFO - Job 1538: Subtask entry_task
[2022-08-29 15:48:11,552] {logging_mixin.py:109} INFO - Running <TaskInstance: h1b_lca_data_v3.entry_task scheduled__2021-01-10T06:00:00+00:00 [running]> on host efb24f1b01c6
[2022-08-29 15:48:11,678] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=h1b_lca_data_v3
AIRFLOW_CTX_TASK_ID=entry_task
AIRFLOW_CTX_EXECUTION_DATE=2021-01-10T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-10T06:00:00+00:00
[2022-08-29 15:48:11,682] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-29 15:48:11,684] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2022-08-29 15:48:11,704] {subprocess.py:85} INFO - Output:
[2022-08-29 15:48:11,708] {subprocess.py:89} INFO - 1
[2022-08-29 15:48:11,710] {subprocess.py:93} INFO - Command exited with return code 0
[2022-08-29 15:48:11,743] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=h1b_lca_data_v3, task_id=entry_task, execution_date=20210110T060000, start_date=20220829T154811, end_date=20220829T154811
[2022-08-29 15:48:11,796] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-29 15:48:11,839] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
